libs/components/datetime/src/lib/modules/datepicker/monthpicker.component.html

```diff
@@ -29,10 +29,6 @@
           type="button"
           class="sky-btn sky-btn-default sky-btn-sm sky-datepicker-calendar-title"
           (click)="datepicker.toggleModeCalendar($event)"
-          [disabled]="datepicker.datepickerMode === maxMode"
-          [ngClass]="{
-            'sky-btn-disabled': datepicker.datepickerMode === maxMode
-          }"
```

Comment:
So I added this back in this commit and fixed an issue where `maxMode` should have been `datepicker.maxMode`. However, this is somewhat dead code today as we alway have `maxMode` set to `year` and so this will never be hit. But in the spirit of not being destructive - I added it back.